doc/src/manual/conversion-and-promotion.md

```diff
@@ -233,11 +241,18 @@ the most typical "clever" application being the definition of catch-all methods
 like the arithmetic operators `+`, `-`, `*` and `/`. Here are some of the catch-all method definitions
 given in [promotion.jl](https://github.com/JuliaLang/julia/blob/master/base/promotion.jl):
 
-```julia
-+(x::Number, y::Number) = +(promote(x,y)...)
--(x::Number, y::Number) = -(promote(x,y)...)
-*(x::Number, y::Number) = *(promote(x,y)...)
-/(x::Number, y::Number) = /(promote(x,y)...)
+```jldoctest
```

Comment:
This is a bit weird to have as a doctest for similar reasons as discussed (overwriting Base methods). Hmm.